Sloth.Web/Controllers/TransactionsController.cs

```diff
@@ -233,6 +237,43 @@ public async Task<IActionResult> CreateReversal(string id)
             return RedirectToAction("Details", new { id = reversal.Id });
         }
 
+        [HttpPost]
+        public async Task<IActionResult> CallWebHook(string id)
+        {
+            var transaction = await DbContext.Transactions
+                .Include(t => t.Source)
+                .ThenInclude(s => s.Team)
+                .SingleOrDefaultAsync(t => t.Id == id);
+
+            if (transaction == null)
+            {
+                return NotFound();
+            }
+
+            if (transaction.Source.Team.Slug != TeamSlug)
+            {
+                return Forbid();
+            }
+
+            var hasWebhooks = await DbContext.WebHooks.AnyAsync(w => w.Team.Slug == TeamSlug);
+
+            if (!hasWebhooks)
+            {
+                return NotFound();
+            }
+
+            await WebHookService.SendBankReconcileWebHook(transaction.Source.Team, new BankReconcileWebHookPayload()
```

Comment:
@sprucely Does this return a value we can display?

Reply:
It simply pops up an alert with english version of the http response, `error: Not Found`, `error: Forbidden`, etc...

https://github.com/ucdavis/Sloth/blob/ee25910953e889ff0875080c87e0357d8f547efc/Sloth.Web/Views/Transactions/Index.cshtml#L76-L86

Reply:
The actual service logs responses, which makes sense as it is normally called from the job.
But it would be nice if possible to show specific feedback to the user from the returned results.

But not a big deal.